### ISSUE TITLE
After Rjb::load code returns zero when convert BigDecimal into float 

### ISSUE THREAD

**Issue by correalucas** [arton/rjb#28]
Hi, 
When i try to convert BigDecimal into float number, i get zero in result
Below, the versions of my enviroment and the test: 

ruby --version => ruby 1.9.3p484 (2013-11-22 revision 43786) [i686-linux]

java -version => java version "1.7.0_45" - Java(TM) SE Runtime Environment (build 1.7.0_45-b18) - Java HotSpot(TM) Client VM (build 24.45-b08, mixed mode)

Gems => rjb (1.4.8), bigdecimal (1.1.0)
<code>
irb => 
1.9.3-p484 :001 > require 'bigdecimal'
 => true
1.9.3-p484 :002 > require 'rjb'
 => true
1.9.3-p484 :003 > n = BigDecimal.new('99.99')
 => #< BigDecimal:8b9a304,'0.9999E2',18(18) >
1.9.3-p484 :004 > n.to_f
 => 99.99
1.9.3-p484 :005 > Rjb::load '.', ['-Xmx1024m']
 => nil
1.9.3-p484 :006 > n = BigDecimal.new('99.99')
 => #< BigDecimal:8a26c5c,'0.9999E2',18(18) > 
1.9.3-p484 :007 > n.to_f
 => 0.0
</code>
I await return,
Thanks.

**Comment by gregoriokusowski** [comment]
Hi there guys.

I'm having this issue on a single server in a cluster of 3.
Do you guys think it's OK to include a gem that handles what the stdlib already provides?

I'm taking a look at configs like locale/etc on all servers, because it's working correctly on 2 of the 3.
I'll let you guys know if I discover something.

Oh, I'm using MRI 2.1.6 and rjb 1.4.8.

**Comment by arton** [comment]
Hi gregoriokusowski
Did you install extra gem on cluster #3?
I wonder if you could check .gemspec and rbenv (or somthing of env controller) settings.

**Comment by gregoriokusowski** [comment]
@arton,
nope, they all have the same gems, and we don't have `bigdecimal` in our Gemfile.
We're using rvm.

**Comment by gregoriokusowski** [comment]
Solved:
The third machine was displaying this warning `perl: warning: Setting locale failed.`.
After fixing that, I've reinstalled the Ruby 2.1.6 and the Rjb gem, but the problem persisted (yesterday).
Today, in order to make a few more tests to isolate the test, I've logged into the machine and the problem isn't happening anymore. (My plan was to compare env dumps between those machines)
I think that all I needed to do is to open a new session on that server in order to make it work.

Our unicorn is up and running, and now displaying more than zeros for our users ;)

TL;DR: If you are facing this problem, check your locale settings, and spin up a new session after each change to validate.

Thank you guys.

**Comment by arton** [comment]
Hi Gregório 
Thank you for your report for solving!
It's great help for us.
